specifications/active/aRMT-1.0.0.yml

```diff
@@ -106,3 +106,975 @@ data:
             value: 2
           - text: Nearly every day
             value: 3
+
+  - type: ESM
+    topic: questionnaire_esm
+    value_schema: .active.questionnaire.Questionnaire
+    questions:
+      - lead: Null
+        content: I slept well
```

Comment:
Is it intentional that statements like `I slept well` are not punctuated, and questions like `What am I doing (just before the beep)?`  are?

Reply:
I don't know about this. This schema is provided by @kerzmaximilian . Maybe he can comment better on this than me. I guess since this is just a statement rather than a question thus no `?`